Tests/StatsdClientTests/Stub|Mock|Fake|Spy/SpyScheme.swift

```diff
@@ -11,12 +11,12 @@ import Foundation
 
 class SpyScheme: FlushScheme {
 
-    var isStartCall = false
+    var isCallStrated = false
```

Comment:
`Strated`?

Reply:
Did you mean `Started`?

Reply:
My fault. I didn't realize it. 
It's reason why I'm studying English =))